napari/components/_tests/test_viewer_keybindings.py

```diff
@@ -0,0 +1,27 @@
+from napari.components._viewer_key_bindings import toggle_theme
+from napari.settings import get_settings
+from napari.utils.theme import available_themes
+
+
+def test_theme_toggle_keybinding(make_napari_viewer):
+    viewer = make_napari_viewer()
```

Comment:
maybe use `ViewerModel` instead of `make_napari_viewer` or move this test to `_qt` directory.

Reply:
Is this the way to adress:
>Hmm, for the headless test it's something related to qt:
https://github.com/napari/napari/actions/runs/3452731873/jobs/5766326348#step:8:243
I guess this test, which needs a viewer (I think), should be skipped on headless? Not sure how to do that!

If so, which is better?

Reply:
it depends if the logic of sync theme between settings and viewer is only in `ViewerModel` or is connected with `Viewer`

Reply:
ViewerModel worked, I can't say I totally get why, but it works!

Reply:
Because the ViewerModel itself does not depend on qt.